Adamant/Modules/Settings/Notifications/NotificationsView.swift

```diff
@@ -0,0 +1,234 @@
+        } header: {
+            Text(viewModel.notificationsTitle)
+        }
+    }
+    
+    func messageSoundSection() -> some View {
+        Section {
+            Button(action: {
+                viewModel.presentNotificationSoundsPicker()
+            }, label: {
+                HStack {
+                    Text(soundTitle)
+                    Spacer()
+                    Text(viewModel.notificationSound.localized)
+                        .foregroundColor(.gray)
+                    NavigationLink(destination: { EmptyView() }, label: { EmptyView() }).fixedSize()
+                }
+            })
+        } header: {
+            Text(messagesHeader)
+        }
+    }
+    
+    func messageReactionsSection() -> some View {
+        Section {
+            Button(action: {
+                viewModel.presentReactionNotificationSoundsPicker()
+            }, label: {
+                HStack {
+                    Text(soundTitle)
+                    Spacer()
+                    Text(viewModel.notificationReactionSound.localized)
+                        .foregroundColor(.gray)
+                    NavigationLink(destination: { EmptyView() }, label: { EmptyView() }).fixedSize()
+                }
+            })
+        } header: {
+            Text(reactionsHeader)
+        }
+    }
+    
+    func inAppNotificationsSection() -> some View {
+        Section {
+            Toggle(isOn: $viewModel.inAppSounds) {
+                Text(soundsTitle)
+            }
+            .tint(.init(uiColor: .adamant.active))
+            .onChange(of: viewModel.inAppSounds) { value in
+                viewModel.applyInAppSounds(value: value)
+            }
+            
+            Toggle(isOn: $viewModel.inAppVibrate) {
+                Text(vibrateTitle)
+            }
+            .tint(.init(uiColor: .adamant.active))
+            .onChange(of: viewModel.inAppVibrate) { value in
+                viewModel.applyInAppVibrate(value: value)
+            }
+            
+            Toggle(isOn: $viewModel.inAppToasts) {
+                Text(toastsTitle)
+            }
+            .tint(.init(uiColor: .adamant.active))
+            .onChange(of: viewModel.inAppToasts) { value in
+                viewModel.applyInAppToasts(value: value)
+            }
+        } header: {
+            Text(inAppNotifications)
+        }
+    }
+    
+    func settingsSection() -> some View {
+        Section {
+            Button(action: {
+                viewModel.openAppSettings()
+            }, label: {
+                HStack {
+                    Text(settingsHeader)
+                    Spacer()
+                    NavigationLink(destination: { EmptyView() }, label: { EmptyView() }).fixedSize()
+                }
+            })
+        } header: {
+            Text(settingsHeader)
+        }
+    }
+    
+    func moreDetailsSection() -> some View {
+        Section {
+            if let attributedString = viewModel.parseMarkdown(descriptionText) {
```

Comment:
I don't like the view doing this job with processing Strings before presenting. Let's do everything related to processing in the view model. View is only presenting the view model's state.